syz-verifier/main.go

```diff
@@ -493,7 +489,7 @@ func (srv *RPCServer) newProgram(poolIdx, vmIdx int) ([]byte, int) {
 			prog:       prog,
 			idx:        progIdx,
 			serialized: prog.Serialize(),
-			res:        make([]*verf.Result, 0),
+			res:        make([]*Result, 0),
```

Comment:
This line is not necessary. A zero value for a slice is an empty slice already. Thus make with 0 size and capacity is generally not used.